rsbuild/react-electron-template/electron-builder.js

```diff
@@ -0,0 +1,79 @@
+const Pakceage = require('./package.json')
+const {resolve} = require("path")
+
+const versionArr = Pakceage.version.split('-')
+const bundleShortVersion = versionArr[0]
+const bundleVersion = versionArr[1]
+
+const config = {
+    asar: true,
```

Comment:
can you use Biome to format the code? thanks

Reply:
ok, fixed it;